controllers

```diff
@@ -77,9 +77,24 @@ function getArticles (req, res, next) {
         res.status(200).send({ article: updatedArticle });
       })
       .catch((err) => {
-        console.log(err)
         next(err);
       });
   }
+
+  function deleteComment(req, res, next) {
+    const { comment_id } = req.params;
+  
+    deleteCommentById(comment_id)
+      .then(() => {
+        res.status(204).end();
+      })
+      .catch((err) => {
+        if (err.message === 'Comment not found') {
+          res.status(404).send({ msg: err.message });
+        } else {
```

Comment:
This logic could be moved to an error handler in _app.js_ to clean up the catch block